packages/cli/src/commands/composite/deploy.ts

```diff
@@ -17,13 +17,26 @@ export default class CompositeDeploy extends Command<
     }),
   }
 
+  static flags = {
+    ...Command.flags,
+    index: Flags.boolean({
+      char: 'i',
+      description: 'Adds the composite to indexing on the ceramic node',
+      default: true,
```

Comment:
don't you need to set `allowNo` to enable the `--no-index` behavior?

Reply:
It doesn't seem like it from the tests. Oclif just seems to support it automatically.